vm/vmStmt.go

```diff
@@ -618,6 +618,99 @@ func (runInfo *runInfoStruct) runSingleStmt() {
 		}
 		runInfo.rv = nilValue
 
+	// SelectStmt
+	case *ast.SelectStmt:
+		e := runInfo.env
+		runInfo.env = e.NewEnv()
```

Comment:
Should select be in a new env?

Reply:
Can you give me some information about what should and should not be in a new env ?
I inspired myself from the `SwitchStmt` which has a new env.
https://github.com/mattn/anko/blob/57d52a5bf01245ecdfd175f67e8bbfbe663c64a8/vm/vmStmt.go#L715-L717

Reply:
Each switch expression needed to be in a new scope (a new env) so that one switch expression did not affect another switch express. I am not sure if that is needed here.